### PR TITLE
Fix validation for breakdowns

### PR DESCRIPTION
**Pull request by timgl**
## Changes

Breakdowns were erroring as we were sending 

@pauldambra `breakdowns` (with an `s`) seems quite brittle just trying it out. I've documented it as 'experimental' which I assume is right?

## How did you test this code?

unit tests

### CODE REVIEW

File: posthog/api/insight_serializers.py [hunk 1]
```diff
@@ -189,3 +224,7 @@ class FunnelSerializer(GenericInsightsSerializer, BreakdownMixin):
     funnel_window_days = serializers.IntegerField(
         help_text="(DEPRECATED) Funnel window size in days.", required=False, default=14
     )
+    breakdowns = BreakdownField(
+        required=False,
+        help_text='**Experimental**. Alternative to `breakdown`, where you can pass through an object with different types of breakdowns.\nFor example: `[{property: 291, type: "cohort"}, {property: "email", type: "person"}]`',
```
Comment (pauldambra) [inline posthog/api/insight_serializers.py:229]: Yep, deffo experimental where I last touched it 

(I don't know if @hazzadous moved it on at all although he might have)

It was aimed at allowing mixed breakdowns but the underlying queries assume all breakdowns are the same time... 

so `[{property:"email", type: "person", value: "a", operator: "contains"}, {property: "email", type: "person", value: "b", operator: "contains"}]`

(assuming I've remembered the operator names correctly)

**Review by pauldambra — APPROVED** [review]

File: posthog/api/insight_serializers.py (hunk 1 above)
Comment (hazzadous) [reply posthog/api/insight_serializers.py:229]: I didn't change to support mixed breakdowns in the implementation.

Comment (timgl) [reply posthog/api/insight_serializers.py:229]: Ah, in that case should we roll back this feature flag? It's quite buggy and currently rolled out to beta users: https://app.posthog.com/feature_flags/1119.

I'll remove this field

Comment (hazzadous) [reply posthog/api/insight_serializers.py]: Definitely if it's only delivering pain

**Pull request merged**